mavlink/modules/flight_controller.py

```diff
@@ -95,3 +95,21 @@ def get_home_location(self, timeout: float) -> "tuple[bool, drone_odometry.Drone
             return False, None
 
         return True, location
+
+    def drone_travelling_to_final_waypoint(self) -> "tuple[bool, bool | None]":
+        """
+        Returns whether the drone is travelling to the final waypoint in the mission.
+        """
+        waypoint_count = self.drone.commands.count
+        current_waypoint = self.drone.commands.next
```

Comment:
If a new mission is uploaded to the drone, do the Dronekit-Python variables remain accurate?

Are these variables guaranteed to be integers (e.g. what is the type when the drone is first turned on)?

Reply:
Yes, the variables remain accurate. The variables are guaranteed to be integers.